core/trino-main/src/main/java/io/trino/sql/analyzer/StatementAnalyzer.java

```diff
@@ -535,6 +535,7 @@ protected Scope visitInsert(Insert insert, Optional<Scope> scope)
             List<ColumnSchema> columns = tableSchema.getColumns().stream()
                     .filter(column -> !column.isHidden())
                     .collect(toImmutableList());
+            List<String> checkConstraints = tableSchema.getTableSchema().getCheckConstraints();
```

Comment:
Please apply changes to UPDATE and MERGE too.

Reply:
Can we separate PR for those statements?

Reply:
I think we can.

Reply:
I will add comments with link to #15411